Apr 21, 2017 9:42:10 AM org.xnio.Xnio <clinit>
INFO: XNIO version 3.4.0.Final
Apr 21, 2017 9:42:10 AM org.xnio.nio.NioXnio <clinit>
INFO: XNIO NIO Implementation Version 3.4.0.Final
Apr 21, 2017 9:42:11 AM org.jboss.remoting3.EndpointImpl <clinit>
INFO: JBoss Remoting version 4.0.21.Final
jndiname is : Remote/EJB-Remote-Demo-ejb-1.0/CalculatorBean!com.maggioni.Stateless2.RemoteCalculator
Exception in thread "main" javax.naming.CommunicationException: Failed to connect to any server. Servers tried: [http-remoting://env-2627208.jelastic.dogado.eu:4447 (java.net.ConnectException: Connection refused: no further information)]
	at org.jboss.naming.remote.client.HaRemoteNamingStore.failOverSequence(HaRemoteNamingStore.java:244)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingStore(HaRemoteNamingStore.java:149)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:130)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at com.maggioni.client.RemoteEJBClient.lookupRemoteStatelessCalculator(RemoteEJBClient.java:78)
	at com.maggioni.client.RemoteEJBClient.invokeStatelessBean(RemoteEJBClient.java:22)
	at com.maggioni.client.RemoteEJBClient.main(RemoteEJBClient.java:16)